arctic-report-card/ARCRC.ttl

```diff
@@ -470,6 +470,163 @@ odo:ARCRC_00000025 rdf:type owl:Class ;
+                   obo:IAO_0000115 "total number of unique ships based on daily observation" ;
+                   obo:IAO_0000119 "Unique ship days as defined in the Arctic Report Card 2022 Essay at https://arctic.noaa.gov/report-card/report-card-2022/satellite-record-of-pan-arctic-maritime-ship-traffic/" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Other Indicator, 'Satellite Record of Pan-Arctic Maritime Ship Traffic'" ;
+                   rdfs:label "unique ship days"@en .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000027
+odo:ARCRC_00000027 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "model or type of ship/vessel observed" ;
+                   obo:IAO_0000119 "ship type as defined in the Arctic Report Card 2022 Essay at https://arctic.noaa.gov/report-card/report-card-2022/satellite-record-of-pan-arctic-maritime-ship-traffic/" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Other Indicator, 'Satellite Record of Pan-Arctic Maritime Ship Traffic'" ;
+                   rdfs:label "ship type"@en .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000028
+odo:ARCRC_00000028 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "flag of state for national identification" ;
+                   obo:IAO_0000119 "ship flag as defined in the Arctic Report Card 2022 Essay at https://arctic.noaa.gov/report-card/report-card-2022/satellite-record-of-pan-arctic-maritime-ship-traffic/" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Other Indicator, 'Satellite Record of Pan-Arctic Maritime Ship Traffic'" ;
+                   rdfs:label "ship flag"@en .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000029
+odo:ARCRC_00000029 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "measurement of maximum tonnage of ships" ;
+                   obo:IAO_0000119 "ship size as defined in the Arctic Report Card 2022 Essay at https://arctic.noaa.gov/report-card/report-card-2022/satellite-record-of-pan-arctic-maritime-ship-traffic/" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Other Indicator, 'Satellite Record of Pan-Arctic Maritime Ship Traffic'" ;
+                   rdfs:label "ship size"@en .         
```

Comment:
Can we link these to existing ontology terms? There are a lot of ship/ocean vessel vocabularies I think. e.g., see the vocabs in use at BCO-DMO.

Reply:
Added narrowMatch and relatedMatches to the ship key variables